oracle/oracle_test.go

```diff
@@ -93,6 +97,175 @@ func Test_Oracle_ManualSubscription(t *testing.T) {
 
 // TODO: Mix manual and automatic subscriptions
 
+// Simulates 100 slots with "AdvanceStateToNextSlot". Each slot is configured randomly with a
+// new sub, unsub or donation. The block proposed can be okproposal, missed or wrongfee.
+// these are all randomly set each block
+var numIterations = flag.Int("iterations", 100, "Number of iterations for the test")
```

Comment:
lets remove this flag. just have it as a single hardcoded value at the begning of the test. `numblocks := xxx`

Reply:
Done